.github/workflows/android-dev-orange-deploy.yaml

```diff
@@ -0,0 +1,53 @@
+name: Android dev-orange deploy
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - synchronize
+      - labeled
+  schedule:
+    - cron: "13 12 * * MON"
+
+jobs:
+  check:
+    name: Check if build is necessary
+    runs-on: ubuntu-latest
+    steps:
+      - id: check
+        name: Check
+        run: |
+          if ${{ github.event_name == 'schedule' }}; then
+            echo "should-run=$(gh pr list --label 'deploy to android dev-orange' --json id --jq 'length == 0')" | tee -a "$GITHUB_OUTPUT"
+          elif ${{ github.event_name == 'pull_request' }}; then
+            echo "should-run=${{ contains(github.event.pull_request.labels.*.name, 'deploy to android dev-orange') }}" | tee -a "$GITHUB_OUTPUT"
+          else
+            echo "should-run=true" | tee -a "$GITHUB_OUTPUT"
+          fi
+        env:
+          GH_REPO: ${{ github.repository }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    outputs:
+      should-run: ${{ steps.check.outputs.should-run }}
+  build-android:
+    name: Build for Android
+    needs: check
+    if: ${{ needs.check.outputs.should-run == 'true' }}
+    permissions:
+      id-token: write
+      contents: read
+      pull-requests: write
+    uses: ./.github/workflows/build-android.yaml
+    with:
+      build-type: "Release"
+      deploy: true
+      flavor: "DevOrange"
+      sentry-environment: "dev-orange"
+    secrets:
+      AWS_ROLE_ARN: ${{ secrets.AWS_ROLE_ARN }}
+      FIREBASE_KEY: ${{ secrets.FIREBASE_KEY }}
+      GCP_WORKLOAD_IDENTITY_PROVIDER: ${{ secrets.GCP_WORKLOAD_IDENTITY_PROVIDER }}
+      GCP_SERVICE_ACCOUNT: ${{ secrets.GCP_SERVICE_ACCOUNT }}
+      GOOGLE_APP_ID_ANDROID: ${{ secrets.GOOGLE_APP_ID_ANDROID_DEV_ORANGE }}
```

Comment:
This secret and its iOS cousin will need to be populated - I’ve created the apps in Firebase but I don’t have access to the GitHub repo settings.

Reply:
Added!